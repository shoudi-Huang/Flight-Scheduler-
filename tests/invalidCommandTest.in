helt

hElps
fligh
flightes 0
locate
locasions
SCHEDUL
Depareture
arRiviEd
Travell
exit
