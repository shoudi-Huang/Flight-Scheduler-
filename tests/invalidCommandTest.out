User: Invalid command. Type 'help' for a list of commands.

User: Invalid command. Type 'help' for a list of commands.

User: Invalid command. Type 'help' for a list of commands.

User: Invalid command. Type 'help' for a list of commands.

User: Invalid command. Type 'help' for a list of commands.

User: Invalid command. Type 'help' for a list of commands.

User: Invalid command. Type 'help' for a list of commands.

User: Invalid command. Type 'help' for a list of commands.

User: Invalid command. Type 'help' for a list of commands.

User: Invalid command. Type 'help' for a list of commands.

User: Invalid command. Type 'help' for a list of commands.

User: Application closed.
